app/Http/Controllers/IndustryCategoryController.php

```diff
@@ -71,9 +74,15 @@ public function edit(IndustryCategory $industryCategory)
      */
     public function update(IndustryCategoryRequest $request, IndustryCategory $industryCategory)
     {
-        $industryCategory->update($request->all());
-        return redirect(route("industry-categories.index", $industryCategory))
-            ->with(session()->flash('success','Industry category is Updated successfully .'));;
+        if (IndustryCategory::where('name', $request->name)->where('id', '!=', $industryCategory->id)->count() == 0) {
+            IndustryCategory::where('id', $industryCategory->id)->update(['name' => $request->name]);
+             return redirect()->route('industry-categories.index')
+                        ->with(session()->flash('success','Industry category is Updated successfully .'));
+        }
+        else{
+            $err['name']='This is already exist';
```

Comment:
Rename it to
This category already exists.

Reply:
done